test/org/sagebionetworks/bridge/dynamodb/DynamoAppConfigTest.java

```diff
@@ -66,32 +91,39 @@ public void canSerialize() throws Exception {
         appConfig.setModifiedOn(TIMESTAMP.getMillis());
         appConfig.setSurveyReferences(SURVEY_REFS);
         appConfig.setSchemaReferences(SCHEMA_REFS);
+        appConfig.setConfigReferences(CONFIG_REFS);
+        appConfig.setConfigElements(ImmutableMap.of("config1", TestUtils.getClientData()));
         appConfig.setClientData(clientData);
         appConfig.setVersion(3L);
         appConfig.setDeleted(true);
         
         Set<String> fields = Sets.newHashSet("criteria", "label", "createdOn", "modifiedOn", "clientData",
-                "surveyReferences", "schemaReferences", "version", "type", "deleted");
+                "surveyReferences", "schemaReferences", "configReferences", "configElements", "version", 
+                "type", "deleted");
                 
         JsonNode node = BridgeObjectMapper.get().valueToTree(appConfig);
         assertEquals(fields, Sets.newHashSet(node.fieldNames()));
         
+        TestUtils.assertNode(clientData, node.get("clientData"));
         assertEquals("AppConfig", node.get("type").asText());
         assertEquals(TIMESTAMP.toString(), node.get("createdOn").textValue());
         assertEquals(TIMESTAMP.toString(), node.get("modifiedOn").textValue());
```

Comment:
I think there's still a misunderstanding here. What we need is to verify the JSON representation of the AppConfig object as a whole, not just the the clientData.